client/layout/sidebar/footer.jsx

```diff
@@ -15,8 +15,11 @@ import HappychatButton from 'components/happychat/button';
 const SidebarFooter = ( { translate, children } ) => (
 	<div className="sidebar__footer">
 		{ children }
-		<Button compact borderless href="/help">
-			<Gridicon icon="help-outline" /> { translate( 'Help' ) }
+		<Button borderless href={ config( 'signup_url' ) + '?ref=calypso-selector' }>
```

Comment:
I hear you — upon reflection, perhaps "Add New Site" shouldn't be that prominent. Perhaps we should keep the current behavior:

- Add New Site only on the site selector (perhaps redesigned with bigger icon and footer bar to match this new style)
- Add New Site permanently footer visible in My Sites, for single-site users. 

However I disagree that "Add New Post" should be added to the footer. Doing so is essentially admitting defeat with this button:

![screen shot 2016-10-19 at 13 11 31](https://cloud.githubusercontent.com/assets/1204802/19516712/93e416fa-95fd-11e6-939f-b7aa53a1d2f1.png)

If that button doesn't work as intended, we could consider beefing it up at desktop breakpoints:

![screen shot 2016-10-19 at 13 12 49](https://cloud.githubusercontent.com/assets/1204802/19516757/cf119176-95fd-11e6-9641-77592e0c1240.png)

But having multiple buttons that do the same almost always makes the user pose the question, "which one is the right one?"